### PR TITLE
[TACHYON-1175] util functions for predefined streamOptions

### PR DESCRIPTION
**Pull request by siyangy**
https://tachyon.atlassian.net/browse/TACHYON-1175

### CODE REVIEW

File: integration-tests/src/test/java/tachyon/client/StreamOptionUtils.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package tachyon.client;
```
Comment (yupeng9) [inline integration-tests/src/test/java/tachyon/client/StreamOptionUtils.java]: The build fails because of missing license header. Can you add it?

**Comment by yupeng9** [comment]
@siyangy , thanks for the PR! Going forward, could you keep the git history (i.e. avoid git commit --amend)? So that the comments and reviews can be preserved. Thanks.

File: integration-tests/src/test/java/tachyon/client/StreamOptionUtils.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+import tachyon.client.file.options.InStreamOptions;
+import tachyon.client.file.options.OutStreamOptions;
+import tachyon.conf.TachyonConf;
+import tachyon.util.network.NetworkAddressUtils;
+
+public class StreamOptionUtils {
```
Comment (yupeng9) [inline integration-tests/src/test/java/tachyon/client/StreamOptionUtils.java]: This class can be final.

File: integration-tests/src/test/java/tachyon/client/StreamOptionUtils.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  private StreamOptionUtils() {
+    // not intended for instantiation
+  }
+
+  public static OutStreamOptions getOutStreamOptionsWriteBoth(TachyonConf conf) {
```
Comment (yupeng9) [inline integration-tests/src/test/java/tachyon/client/StreamOptionUtils.java]: Could you add javadocs for all the public methods?

File: integration-tests/src/test/java/tachyon/client/StreamOptionUtils.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    return new OutStreamOptions.Builder(conf).setTachyonStorageType(TachyonStorageType.STORE)
+               .setUnderStorageType(UnderStorageType.SYNC_PERSIST).build();
+  }
+
+  public static OutStreamOptions getOutStreamOptionsWriteTachyon(TachyonConf conf) {
```
Comment (yupeng9) [inline integration-tests/src/test/java/tachyon/client/StreamOptionUtils.java]: The options look like WriteType, so perhaps we can have a factory that takes the WriteType as argument? @apc999, what do you think?

**Comment by gpang** [comment]
Do these convenience methods only make sense in test packages, and not in OutStreamOptions() itself?

**Comment by siyangy** [comment]
The original ticket is about test, and looks like it's less useful outside of test packages, but defer to @apc999.

**Comment by gpang** [comment]
Ok, maybe test module makes more sense.

Most of the convenience methods are essentially ReadType and WriteType, which I think would be nice to use. However, the "local" one is the only one that is different.

**Comment by apc999** [comment]
+1 LGTM. Thanks for the patch! @siyangy  it definitely improves the code readability.

**Comment by siyangy** [comment]
is there anything I need to do here before merge?

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**